### PR TITLE
Rebind fix

### PR DESCRIPTION
**Pull request by rdaly525**

### CODE REVIEW

File: peak/peak.py [hunk 1]
```diff
@@ -1,34 +1,29 @@
...
-        return T
-    elif issubclass(T,Sum):
-        if len(T.mro()) ==3: #This was constructed directly from Sum[]
-            return Sum[[rebind_type(t,family) for t in T.fields]]
-        else: #Construced by inhereting from Sum
-            raise NotImplementedError("NYI")
-    else: #a Non-ADT class
+    elif issubclass(T,(Product,Sum)):
+        return _rebind_bv(T)
+    elif issubclass(T,(AbstractBitVector,AbstractBit)):
```
Comment (cdonovick) [inline peak/peak.py]: why not 
```
elif issubclass(T, AbstractBitVector):
    return rebind_bitvector(T, AbstractBitVector, family.BitVector)
elif issubclass(T, AbstractBit):
    return family.Bit
```

Comment (cdonovick) [reply peak/peak.py]: Or
```
elif issubclass(T, AbstractBitVector):
    if T.is_sized:
        return family.BitVector[t.size]
    else:
        return family.BitVector 
elif issubclass(T, AbstractBit):
    return family.Bit
```

Comment (rdaly525) [reply peak/peak.py]: Didnt know that rebind_bitvector worked on BitVectors

Comment (rdaly525) [reply peak/peak.py]: Fixed

**Pull request merged**